cli/azd/pkg/azure/arm_template.go

```diff
@@ -143,6 +143,7 @@ type AzdMetadata struct {
 	Type               *AzdMetadataType `json:"type,omitempty"`
 	AutoGenerateConfig *AutoGenInput    `json:"config,omitempty"`
 	DefaultValueExpr   *string          `json:"defaultValueExpr,omitempty"`
+	DefaultLocation    *string          `json:"defaultLocation,omitempty"`
```

Comment:
Would it be possible to name this default instead of default location?

Reply:
yes, that makes sense